src/frontend/src/utils/gantt.utils.ts

```diff
@@ -480,28 +480,13 @@ export const GanttWorkPackageStageColorPipe: (stage: WorkPackageStage | undefine
       case WorkPackageStage.Install:
         return pink[800];
       case WorkPackageStage.Testing:
-        return yellow[800];
+        return '#2d6b77';
       default:
         return grey[500];
     }
   }
 };
 
-// maps stage to the desired text color
-export const GanttWorkPackageTextColorPipe: (stage: WorkPackageStage | undefined) => string = (stage) => {
```

Comment:
I would actually just make this a constant like `export const GanttWorkPackageTextColor: string = '#ffffff'`, and then replace the usage of the pipe with this new constant instead. This way if we ever need to change the text color, we only need to change it in one place